packages/flutter/lib/src/widgets/form.dart

```diff
@@ -643,6 +687,20 @@ class FormFieldState<T> extends State<FormField<T>> with RestorationMixin {
     super.deactivate();
   }
 
+  @override
+  void initState() {
+    super.initState();
+    _errorText = RestorableStringN(widget.forceErrorText);
+  }
+
+  @override
+  void didUpdateWidget(FormField<T> oldWidget) {
+    super.didUpdateWidget(oldWidget);
+    if (widget.forceErrorText != oldWidget.forceErrorText) {
+      _errorText.value = widget.forceErrorText;
```

Comment:
What if the validator is returning something nonnull at this point, do you still want to update this? Above I asked a question about whether or not it should be possible to use both at the same time. I think we need to answer that question first before deciding how this should work here.

Reply:
That sounds reasonable if we do allow both parameters to be used at the same time (though I still want to see an example as requested below https://github.com/flutter/flutter/pull/132903/files#r1617904728).